src/scss/_chromecastButton.scss

```diff
@@ -30,3 +31,16 @@ $chromecast-icon-size: 12px !default;
       }
    }
 }
+
+.vjs-chromecast-button.vjs-chromecast-button-lg:not(.vjs-hidden) {
+   // Fits both the icon and the label on the same control
+   display: flex !important;
```

Comment:
Is the `!important` flag really needed here? If so, in general we should try to resolve specificity issues without `!important`. However, I removed it in the demo and `display: flex` had enough specificity on its own.